content/docs/get-started/kubernetes/review-project.md

```diff
@@ -145,18 +145,33 @@ class Program
 This Pulumi program creates an NGINX deployment and exports the name of the deployment.
 
 {{% lang python %}}
+
+> *Note*: As a prerequisite, [install virtualenv](https://virtualenv.pypa.io/en/latest/installation/) to manage project requirements
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python -m venv k-env
```

Comment:
Intentional that this is different than others?  (Python va Python3 and k-env vs venv)

Reply:
fixed